internal/persistence/persister.go

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package persister
```

Comment:
Does the persister deserve its own package? Or can we move it into another package?

Reply:
Thanks, persister refactored on https://github.com/hazelcast/hazelcast-commandline-client/pull/61/commits/9cb4ca319bb38991b8f1d9e773f2bf329b24d6e6